mods/d2k/rules/player.yaml

```diff
@@ -87,7 +87,6 @@ Player:
 		Prerequisites: techlevel.low, techlevel.medium, techlevel.high, techlevel.superweapons
 		Id: unrestricted
 	EnemyWatcher:
-	HarvesterInsurance:
```

Comment:
Uh, this disables it for the whole mod!
You should move this to the shellmaps' rules.yaml.